frontend/src/component/pages/Home.tsx

```diff
@@ -204,7 +222,7 @@ export class Home extends Component<Props, State> {
   render = () => {
     return (
       <div className="home">
-        <ExtPromo />
+        {!this.state.browserHasAppExtension && <ExtPromo />}
```

Comment:
Hey @byliuyang , I don't understand how this could be included as a part of FeatureDecisionService. This component is displayed based on runtime environment of user depending on whether the user has the extension or not. I find this close to displaying a component based on user action rather than a feature which can be toggled, something like if we should show the login-modal based on user's logged-in status. Do you want to move the logic behind finding the extensionStatus to FeatureDecisionService?

Reply:
Hi @rohithbalaji123, you convinced me. How about we do 

```typescript
this.props.chromeExtensionService.isInstalled() && <ExtPromo />
```

Reply:
Yes, this would be nice. I'll make this change.